app/src/main/java/io/pslab/sensors/SensorAPDS9960.java

```diff
@@ -0,0 +1,393 @@
+package io.pslab.sensors;
+
+import android.graphics.Color;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.RelativeLayout;
+import android.widget.SeekBar;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+
+import com.github.mikephil.charting.charts.LineChart;
+import com.github.mikephil.charting.components.Legend;
+import com.github.mikephil.charting.components.XAxis;
+import com.github.mikephil.charting.components.YAxis;
+import com.github.mikephil.charting.data.Entry;
+import com.github.mikephil.charting.data.LineData;
+import com.github.mikephil.charting.data.LineDataSet;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import io.pslab.DataFormatter;
+import io.pslab.R;
+import io.pslab.communication.ScienceLab;
+import io.pslab.communication.peripherals.I2C;
+import io.pslab.communication.sensors.APDS9960;
+import io.pslab.others.ScienceLabCommon;
+
+public class SensorAPDS9960 extends AppCompatActivity {
+    private static final String TAG = "APDS9960";
```

Comment:
Sorry, I was not paying attention when I made the previous suggestion. I usually create TAGs like this:

```suggestion
    private static final String TAG = SensorAPDS9960.class.getSimpleName();
```

The nice thing about it is that if the class name us changed later, the TAG will be updated automatically.